website/content/docs/job-specification/hcl2/index.mdx

```diff
@@ -38,11 +31,28 @@ context from the local environment of the CLI (e.g., files, environment variable
 
 [jobs-api]: /api-docs/jobs
 
-## Syntax
+## JSON Jobs
+
+Since HCL is a superset of JSON, `nomad job run example.json` would attempt to
+parse a JSON job using the HCL parser. However when JSON formatted jobs are
+desired, the preferred approach is to use the [API's JSON
+format][json-jobs-api] and the [`-json` command line flag][run-json]:
```

Comment:
I think having the bit about it being "unspecified" below the example makes it less effective. What if we moved that up here:

```suggestion
Since HCL is a superset of JSON, `nomad job run example.json` will attempt to
parse a JSON job using the HCL parser. However, the JSON format accepted by
the HCL parser is not the same as the [API's JSON format][json-jobs-api]. The 
HCL parser's JSON format is unspecified, so the API format is preferred. You can 
use the API format with the [`-json` command line flag][run-json]:
```

Reply:
Thanks! I like yours much better.